tests/tests_luks_raid.yml

```diff
@@ -0,0 +1,147 @@
+---
+- hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    mount_location: '/opt/test1'
+    storage_use_partitions: true
+    volume_size: '5g'
+    key_file: '/opt/keyfile'
+    passwd: 'yabbadabbadoo'
+
+  tasks:
+    - include_role:
+        name: storage
+
+    - include_tasks: get_unused_disk.yml
+      vars:
+        min_size: "{{ volume_size }}"
+        disks_needed: 2
+        max_return: 3
+
+    ##
+    ## Raid
+    ##
+
+    - name: Test for correct handling of new encrypted pool w/ no key
+      block:
+        - name: Create an encrypted raid volume
+          include_role:
+            name: storage
+          vars:
+            safe_mode: false
+            storage_volumes:
+              - name: foo
+                type: raid
+                disks: "{{ unused_disks }}"
+                raid_level: "raid0"
+                encryption: true
+                volumes:
```

Comment:
How can a storage volume have other volumes under it? Shouldn't it be under `storage_pools`, not `storage_volumes`? @japokorn @dwlehman

Reply:
PR description mentions volumes so I presume that `storage_volumes` should be used here. `storage_volumes` does not support `volumes` attribute though.
I think that what is happening in this case is that `volumes` makes it into the `storage_volumes` dictionary w/o producing an error but then it is never used.

To fix this, the `volumes` contents should be moved directly into `storage_volumes` dictionary:
```
storage_volumes:
  - name: test1
    type: raid
    disks: "{{ unused_disks }}"
    raid_level: "raid0"
    encryption: true
    mount_point: "{{ mount_location }}"
    size: 4g
```

Reply:
Good catch. This is repeated below as well.

Reply:
If it is not used, something is wrong with the result validation in test.

Reply:
> If it is not used, something is wrong with the result validation in test.
Thanks for the review, I have fixed the error. 
BTW, I don't know why it's works before and test pass